lectia-2.js

```diff
@@ -1,3 +1,51 @@
 // https://miorita.scoalaweb.ro/lessons/2-if
 // Scrie un program care să o ajute pe Miorița să ajungă la oricare din pășuni
 // Programul trebuie să funcționeze și cu gard, și fără gard
+reset()
```

Comment:
V-am spus la curs să puneți ; la sfârșitul fiecărei instrucțiuni.
```suggestion
reset();
```